scripts/uninstall.sh

```diff
@@ -21,23 +27,54 @@ TOOLKIT_GITOPS_PATH_STAGING=${TOOLKIT_GITOPS_PATH_STAGING:-staging}
 
 STARTTIME=$(date +%s)
 
+./uninstall-userdemo.sh
+
 oc delete all -l app=gogs
 
 oc delete secret ${TOOLKIT_SECRET} -n openshift-configure
 
 # TODO remove htpasswd entry from OAuth cluster
+# LDS Check with Carlos if the following is correct
+oc replace -f - <<EOF
```

Comment:
We don't want to do it this way, as the OAuth can have multiple entries.

The edit needs to be done in a way that only the entry for toolkit is remove and the other left.

this doesn't have to be tackle in this PR